uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Test/Fabricator/Assignment.php

```diff
@@ -41,11 +43,14 @@ private function setDefaultValues() {
    */
   private static function getOneCaseType() {
     $result = civicrm_api3('CaseType', 'get', [
-      'sequential' => 1,
       'is_active' => 1,
       'options' => ['limit' => 1],
     ]);
 
+    if ($result['count'] == 0) {
+      return CRM_HRCore_Test_Fabricator_CaseType::fabricate()['id'];

```

Comment:
This created a dependency that we don't want to have. T&A should not depend on anything from CiviHR. It's fine to use the API here

Reply:
I'll change it, but you should be aware there are other dependencies on CiviHR in T&A

Reply:
I was not aware of that. It shouldn't have happened